### PR TITLE
 	update installing_cloudforms_on_red_hat_enterprise_linux_openstack_platform deleting downstream references

### PR DESCRIPTION
**Pull request by katafira**
@alexvkcr  @carlosglez7 @sergio-ocon @delaluzparra

I had to rephrase two paragraphs, so I'm gonna leave this as a comment in case. You will also have a changes.txt file in the folder with this message:

dir: installing_cloudforms_on_red_hat_enterprise_linux_openstack_platform
file: configuring_cloudforms.adoc
changes: 

*********************************************************************************************************
line 71
	
	[IMPORTANT]
=======
Before installing an internal database, add a disk to the infrastructure hosting your appliance. See the documentation specific to your infrastructure for instructions on how to add a disk. As a storage disk usually cannot be added while a virtual machine is running, Red Hat recommends adding the disk before starting the appliance. {product-title} only supports installing of an internal VMDB on blank disks. The installation will fail if the disks are not blank.

Red Hat rephrased to:

[IMPORTANT]
=======
Before installing an internal database, add a disk to the infrastructure hosting your appliance. See the documentation specific to your infrastructure for instructions on how to add a disk. As a storage disk usually cannot be added while a virtual machine is running, adding the disk before starting the appliance is recommended. {product-title} only supports installing of an internal VMDB on blank disks. The installation will fail if the disks are not blank.
=======

*********************************************************************************************************

=== Configuring an External Database

The `postgresql.conf` file used with {product-title} databases requires specific settings for correct operation. For example, it must correctly reclaim table space, control session timeouts, and format the PostgreSQL server log for improved system support. Due to these requirements, it is recommended that external {product-title} databases use a `postgresql.conf` file based on the standard file used by the {product-title} Appliance.

Red Hat rephrased to:

=== Configuring an External Database

The `postgresql.conf` file used with {product-title} databases requires specific settings for correct operation. For example, it must correctly reclaim table space, control session timeouts, and format the PostgreSQL server log for improved system support. Due to these requirements, it is recommended that external {product-title} databases use a `postgresql.conf` file based on the standard file used by the {product-title} Appliance.

*********************************************************************************************************

### CODE REVIEW

**Comment by elladeon** [comment]
The changes at line 71 look good to me.

File: installing_cloudforms_on_red_hat_enterprise_linux_openstack_platform/installing_cloudforms.adoc [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+{product-title} is able to be installed and ready to configure in a few quick steps. After downloading {product-title} as a virtual machine image template from the {product-title} Customer Portal, the installation process takes you through the steps of uploading the appliance to a supported virtualization or cloud provider.
+
+[IMPORTANT]
+=======
+After installing the {product-title} Appliance, you must configure the database for {product-title}. See xref:configuring_a_database[].
+=======
+
+=== Obtaining the {product-title} Appliance
+
+. Go to link:https://access.redhat.com[access.redhat.com] and log in to the {product-title} Customer Portal using your customer account details.
```
Comment (elladeon) [inline installing_cloudforms_on_red_hat_enterprise_linux_openstack_platform/installing_cloudforms.adoc]: This is the wrong link and wrong procedure. The downstream appliances are available here:
http://manageiq.org/download/

Comment (katafira) [inline installing_cloudforms_on_red_hat_enterprise_linux_openstack_platform/installing_cloudforms.adoc]: So should I change the link on line 13 with the one you posted?

Comment (chessbyte) [inline installing_cloudforms_on_red_hat_enterprise_linux_openstack_platform/installing_cloudforms.adoc]: @elladeon would you please respond, if you can.

**Comment by katafira** [comment]
Closing pr. Changes pr #51

**Pull request closed without merging**